Examples/Tests/Larmor/inputs

```diff
@@ -83,6 +83,6 @@ warpx.do_moving_window = 0
 warpx.do_plasma_injection = 0
 
 # Laser
-warpx.use_laser    = 0
+lasers.nlasers = 0
```

Comment:
Actually, since `0` is the default, I would suggest to simply drop this line (and `# Laser` above)